src/rocm_docs/rocm_docs_theme/sections/header.html

```diff
@@ -44,6 +44,7 @@
                     {{ nav_button("Infinity Hub", "https://www.amd.com/en/technologies/infinity-hub") }}
                     {{ nav_button("Support", theme_repository_url|replace("-docs", "") + "/issues/new/choose") }}
                     {{ nav_button("Feedback", "mailto:rocm-feedback@amd.com") }}
+                    <div id="readthedocs-embed-flyout"></div>
```

Comment:
This gets hidden behind the footer when you scroll down on a page, which looks weird.

Reply:
setting z-index to 9999 on the footer in custom.css seems to fix this

the change will be visible once it's merged